resources/git_credentials.rb

```diff
@@ -8,6 +8,8 @@
 property :secrets_item,    String, default: lazy { node['osl-git']['secrets_item'] }
 property :use_http_path, [true, false], default: true
 
+unified_mode true
+
```

Comment:
Can you please make a separate PR which addresses this first as a Chef 17 update?